src/lib/form/formManagers/accountDetailsFormManager.ts

```diff
@@ -0,0 +1,35 @@
+import { FormSubmission } from "../../formCache";
```

Comment:
this kinda feels like abuse of the formManager, but it does the job so nicely, there may be a further refactoring to do but not a priority for this sprint.

Reply:
ie the argument is clearly **whooo there, there is no form on this page, what are you doing with a form manager?!** but the problem is the validation is tightly coupled to the idea of the form-manager.